src/components/content-header-card/index.tsx

```diff
@@ -0,0 +1,184 @@
+	className,
+	icon,
+	title,
+	attribution,
+	description,
+	note,
+	badges,
+	dropdown,
+	links,
+	buttons,
+}: ContentHeaderCardProps) {
+	const hasLinks = links && links.length > 0
+	const hasButtons = buttons && buttons.length > 0
+	return (
+		<Card
+			elevation="base"
+			className={classNames(s.contentHeaderCard, className)}
+		>
+			<div className={s.cardTop}>
+				{icon && (
+					<IconTileLogo className={s.icon} size="large" productSlug={icon} />
+				)}
+				<div className={s.content}>
+					<div className={s.upper}>
+						<div className={s.left}>
+							<div className={s.titleWrap}>
+								<Heading size={300} weight="bold" level={1} className={s.title}>
+									{title}
+								</Heading>
+								{attribution && (
+									<Text size={100} weight="medium" className={s.attribution}>
+										{attribution}
+									</Text>
+								)}
+							</div>
+							{description && (
+								<Text size={200} className={s.description}>
+									{description}
+								</Text>
+							)}
+						</div>
+						{dropdown && (
+							<DropdownDisclosure
+								listPosition="right"
+								className={s.dropdown}
+								color="secondary"
+								text={dropdown.text}
+							>
+								{dropdown.items.map((item: DropdownItem) => {
+									return (
+										<DropdownDisclosureLinkItem
+											key={item.text}
+											href={item.href}
+										>
+											{item.text}
+										</DropdownDisclosureLinkItem>
+									)
+								})}
+							</DropdownDisclosure>
+						)}
+					</div>
+					{(note || badges) && (
+						<div className={s.lower}>
+							{badges && (
+								<BadgeList
+									className={s.badgeList}
+									type="outlined"
+									size="small"
+									badges={badges}
+								/>
+							)}
+							{note && (
+								<Text size={200} className={s.note}>
+									{note}
+								</Text>
+							)}
+						</div>
+					)}
+				</div>
+			</div>
+			{(hasLinks || hasButtons) && (
+				<div className={s.cardBottom}>
+					{hasLinks && (
+						<ul className={s.linksList}>
+							{links.map((link: Link) => {
+								return (
+									<li key={link.text}>
+										<StandaloneLink
+											text={link.text}
+											icon={link.icon}
+											href={link.href}
+											opensInNewTab={!link.href.startsWith('/')}
+											iconPosition="leading"
+										/>
+									</li>
+								)
+							})}
+						</ul>
+					)}
+					{hasButtons && (
+						<ul
+							className={classNames(s.buttonsList, {
+								[s.buttonsOnly]: !hasLinks,
+							})}
+						>
+							{buttons.map((button: Button, index: number) => {
+								return (
+									<li key={button.text}>
+										<Button
+											onClick={(e) => {
+												e.preventDefault()
+												button.onClick()
+											}}
+											text={button.text}
+											color={
+												button.isPrimary && index === 0
+													? 'primary'
+													: 'secondary'
+											}
```

Comment:
🙋🏻 Do we plan to put checks in place so that only primary button can be rendered in this list?

Reply:
I think that you mean:

>  Do we plan to put checks in place so that only **one** primary button can be rendered in this list?

If that's what you intended: I verify that the `index === 0` above.  This allows the user to opt-in to the first button being blue (but not allowing others to opt-in).

Reply:
Whoops, I did mean _one_ primary button! Thanks for clarifying, my eyes overlooked that `index === 0`. 😆